src/windows/gfx/d3d11-ctx.c

```diff
@@ -46,8 +59,133 @@ static void d3d11_ctx_get_size(struct d3d11_ctx *ctx, uint32_t *width, uint32_t
 	*height = rect.bottom - rect.top;
 }
 
+static bool d3d11_ctx_refresh_window_bounds(struct d3d11_ctx *ctx)
+{
+	bool changed = false;
+
+	RECT window_bounds_new = {0};
+	GetWindowRect(ctx->hwnd, &window_bounds_new);
+
+	const LONG dt_left = window_bounds_new.left - ctx->window_bounds.left;
+	const LONG dt_top = window_bounds_new.top - ctx->window_bounds.top;
+	const LONG dt_right = window_bounds_new.right - ctx->window_bounds.right;
+	const LONG dt_bottom = window_bounds_new.bottom - ctx->window_bounds.bottom;
+
+	changed = dt_left || dt_top || dt_right || dt_bottom;
+
+	ctx->window_bounds = window_bounds_new;
+
+	return changed;
+}
+
+static bool d3d11_ctx_query_hdr_support(struct d3d11_ctx *ctx)
```

Comment:
It may be worth splitting this in two; One function to retrieve the output the app has the greatest overlap with, and the other to determine if that output supports HDR.

This makes the output retrieval accessible for any future work, and means there's less of a big monolithic function going on here

Reply:
Sure